params/config.go

```diff
@@ -719,6 +720,21 @@ func (c *ChainConfig) ElasticityMultiplier() uint64 {
 	return DefaultElasticityMultiplier
 }
 
+// LatestFork returns the latest time-based fork that would be active for the given time.
+func (c *ChainConfig) LatestFork(time uint64) forks.Fork {
+	// Assume last non-time-based fork has passed.
+	london := c.LondonBlock
+
+	switch {
+	case c.IsCancun(london, time):
+		return forks.Cancun
+	case c.IsShanghai(london, time):
+		return forks.Shanghai
+	default:
+		return forks.Paris
+	}
```

Comment:
You could add Prague too?

Reply:
Ping on this. This PR looks mergeable to me, just some minor things to clear out

Reply:
Ping again

Reply:
Hey I'm still waiting for the actual change to be decided upon. The latest PR to engine API is https://github.com/ethereum/execution-apis/pull/498 which is based on our discussion in Istanbul.

Reply:
That PR seems merged. Is there anything holding this PR?

Reply:
done in 626bc7185 - i don't think anything is holding this PR back now